runtime/autoload/provider/pythonx.vim

```diff
@@ -83,8 +83,8 @@ function! s:check_interpreter(prog, major_ver, skip) abort
   " Try to load neovim module, and output Python version.
   " Return codes:
   "   0  Neovim module can be loaded.
-  "   1  Something else went wrong.
```

Comment:
Not sure. I thought it was documenting `s: s:check_interpreter`, but the placement here would be odd.